whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/maintainer/MaintainerNameValidator.java

```diff
@@ -32,6 +33,9 @@
     @Override
     public void validate(final PreparedUpdate update, final UpdateContext updateContext) {
         final RpslObject updatedObject = update.getUpdatedObject();
+        if (Action.CREATE.equals(update.getAction()) && !updatedObject.getKey().toUpperCase().endsWith(MNT_NAME_SUFFIX)) {
```

Comment:
No need for toUpperCase(), CIString implements endsWith()

Reply:
Removed